dist/configs/itwinjs-recommended.js

```diff
@@ -2,27 +2,33 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-/** @type{import("eslint").Linter.BaseConfig} */
-module.exports = {
-  env: {
-    "browser": true
+
+const typescriptEslintPlugin = require("@typescript-eslint/eslint-plugin");
+
+module.exports =
+{
+  languageOptions: {
+    sourceType: "module",
+    parser: require("@typescript-eslint/parser"),
+    parserOptions: {
+      project: "tsconfig.json",
+      ecmaVersion: "latest",
+      ecmaFeatures: {
+        jsx: true,
+        modules: true
+      },
+    },
   },
-  extends: [
-    "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
-  ],
-  parser: "@typescript-eslint/parser",
-  parserOptions: {
-    "project": "tsconfig.json",
-    "sourceType": "module"
+  plugins: {
+    "@typescript-eslint": typescriptEslintPlugin,
+    "import": require("eslint-plugin-import"),
+    "prefer-arrow": require("eslint-plugin-prefer-arrow"),
+    "deprecation": require("eslint-plugin-deprecation"),
+    "@itwin": require("../rules/index")
```

Comment:
hm, I'm a little confused... I thought we came up with a better solution in our call yesterday... but looks like you still need that "rules" property in `rules/index.js`